Unable to connect to Metabase DB w/ Windows 10

Hey,

I'm on Windows, and wanted to give a try at metabase using the .jar, but it seems it can't connect to the metabase DB (on h2 by default).
I've looked around in the issue and just saw #1853 that pops the same error as me, but in my case, I can't even make it run.

I mean, I can start it, go to `localhost:3000` but the loading screen stays (i can't access the create account screens) and in the logs, there are infinite calls to `GET /api/health 503`.

Am I missing something ?
Do I have a way to know what's going on ? 

Here are the logs :

```
$ java -jar metabase.jar
2016-02-04 00:10:14,447 [INFO ] org.eclipse.jetty.util.log :: Logging initialized @1988ms
WARNING: update already refers to: #'clojure.core/update in namespace: clj-http.client, being replaced by: #'clj-http.client/update
2016-02-04 00:10:19,458 [INFO ] metabase.core :: Starting Metabase in STANDALONE mode
2016-02-04 00:10:19,493 [INFO ] metabase.core :: Launching Embedded Jetty Webserver with config:
 {:port 3000}

2016-02-04 00:10:19,514 [INFO ] org.eclipse.jetty.server.Server :: jetty-9.2.z-SNAPSHOT
2016-02-04 00:10:19,544 [INFO ] org.eclipse.jetty.server.ServerConnector :: Started ServerConnector@19e64900{HTTP/1.1}{0.0.0.0:3000}
2016-02-04 00:10:19,545 [INFO ] org.eclipse.jetty.server.Server :: Started @7087ms
2016-02-04 00:10:19,553 [INFO ] metabase.core :: Starting Metabase version v0.14.0 (bbfdf23 HEAD)...
2016-02-04 00:10:19,842 [INFO ] metabase.driver :: loading driver namespace:  metabase.driver.druid
2016-02-04 00:10:19,927 [DEBUG] metabase.driver :: Registered driver :druid.
2016-02-04 00:10:19,938 [INFO ] metabase.driver :: loading driver namespace:  metabase.driver.h2
2016-02-04 00:10:20,068 [DEBUG] metabase.driver :: Registered driver :h2.
2016-02-04 00:10:20,070 [INFO ] metabase.driver :: loading driver namespace:  metabase.driver.mongo
2016-02-04 00:10:21,134 [DEBUG] metabase.driver :: Registered driver :mongo.
2016-02-04 00:10:21,135 [INFO ] metabase.driver :: loading driver namespace:  metabase.driver.mysql
2016-02-04 00:10:21,145 [DEBUG] metabase.driver :: Registered driver :mysql.
2016-02-04 00:10:21,146 [INFO ] metabase.driver :: loading driver namespace:  metabase.driver.postgres
2016-02-04 00:10:21,160 [DEBUG] metabase.driver :: Registered driver :postgres.
2016-02-04 00:10:21,164 [INFO ] metabase.driver :: loading driver namespace:  metabase.driver.redshift
2016-02-04 00:10:21,172 [DEBUG] metabase.driver :: Registered driver :redshift.
2016-02-04 00:10:21,172 [INFO ] metabase.driver :: loading driver namespace:  metabase.driver.sqlite
2016-02-04 00:10:21,182 [DEBUG] metabase.driver :: Registered driver :sqlite.
2016-02-04 00:10:21,183 [INFO ] metabase.driver :: loading driver namespace:  metabase.driver.sqlserver
2016-02-04 00:10:21,194 [DEBUG] metabase.driver :: Registered driver :sqlserver.
2016-02-04 00:10:21,195 [INFO ] metabase.driver :: loading driver namespace:  metabase.driver.sync
2016-02-04 00:10:21,443 [INFO ] metabase.db :: Verifying Database Connection ...
2016-02-04 00:10:26,448 [ERROR] metabase.driver :: Failed to connect to database: Timed out after 5000 milliseconds.
Exception in thread "main" java.lang.AssertionError: Assert failed: Unable to connect to Metabase DB.
(binding [*allow-potentailly-unsafe-connections* true] (require (quote metabase.driver)) ((clojure.core/deref (resolve (quote metabase.driver/can-connect-with-details?))) engine details))
        at metabase.db$verify_db_connection.invoke(db.clj:149)
        at metabase.db$setup_db.doInvoke(db.clj:163)
        at clojure.lang.RestFn.invoke(RestFn.java:421)
        at metabase.core$init_BANG_.invoke(core.clj:130)
        at metabase.core$start_normally.invoke(core.clj:211)
        at metabase.core$_main.doInvoke(core.clj:245)
        at clojure.lang.RestFn.invoke(RestFn.java:397)
        at clojure.lang.AFn.applyToHelper(AFn.java:152)
        at clojure.lang.RestFn.applyTo(RestFn.java:132)
        at metabase.core.main(Unknown Source)
2016-02-04 00:10:42,649 [ERROR] metabase.middleware :: GET /api/health 503 (1 ms)
{:status "initializing", :progress 0.4}

2016-02-04 00:10:43,274 [ERROR] metabase.middleware :: GET /api/health 503 (170 Ás)
{:status "initializing", :progress 0.4}
```

The H2 db is properly created in my current folder : 

```
$ ls metabase.db.h2.db
metabase.db.h2.db
```

Thanks,

Hi I probably have the same problem.

Setup
Windows 10, java 1.8.0_73

Have tried:
1. jar file
2. updated java (had older version -> upgraded to 1.8.0_73)
3. restart windows 
4. tried docker 
5. tried Mozilla, Chrome, Edge

Still I get the following log:

```
$ java -jar metabase.jar
2016-02-09 16:30:44,789 [INFO ] org.eclipse.jetty.util.log :: Logging initialized @2830ms
WARNING: update already refers to: #'clojure.core/update in namespace: clj-http.client, being replaced by: #'clj-http.client/update
2016-02-09 16:30:49,880 [INFO ] metabase.core :: Starting Metabase in STANDALONE mode
2016-02-09 16:30:49,905 [INFO ] metabase.core :: Launching Embedded Jetty Webserver with config:
 {:port 3000}

2016-02-09 16:30:49,926 [INFO ] org.eclipse.jetty.server.Server :: jetty-9.2.z-SNAPSHOT
2016-02-09 16:30:49,957 [INFO ] org.eclipse.jetty.server.ServerConnector :: Started ServerConnector@167c42f{HTTP/1.1}{0.0.0.0:3000}
2016-02-09 16:30:49,957 [INFO ] org.eclipse.jetty.server.Server :: Started @8000ms
2016-02-09 16:30:49,962 [INFO ] metabase.core :: Starting Metabase version v0.14.1 (56b123e release-0.14.1)...
2016-02-09 16:30:50,384 [INFO ] metabase.driver :: loading driver namespace:  metabase.driver.druid
2016-02-09 16:30:50,449 [DEBUG] metabase.driver :: Registered driver :druid.
2016-02-09 16:30:50,460 [INFO ] metabase.driver :: loading driver namespace:  metabase.driver.h2
2016-02-09 16:30:50,624 [DEBUG] metabase.driver :: Registered driver :h2.
2016-02-09 16:30:50,626 [INFO ] metabase.driver :: loading driver namespace:  metabase.driver.mongo
2016-02-09 16:30:51,537 [DEBUG] metabase.driver :: Registered driver :mongo.
2016-02-09 16:30:51,538 [INFO ] metabase.driver :: loading driver namespace:  metabase.driver.mysql
2016-02-09 16:30:51,552 [DEBUG] metabase.driver :: Registered driver :mysql.
2016-02-09 16:30:51,553 [INFO ] metabase.driver :: loading driver namespace:  metabase.driver.postgres
2016-02-09 16:30:51,590 [DEBUG] metabase.driver :: Registered driver :postgres.
2016-02-09 16:30:51,594 [INFO ] metabase.driver :: loading driver namespace:  metabase.driver.redshift
2016-02-09 16:30:51,603 [DEBUG] metabase.driver :: Registered driver :redshift.
2016-02-09 16:30:51,603 [INFO ] metabase.driver :: loading driver namespace:  metabase.driver.sqlite
2016-02-09 16:30:51,615 [DEBUG] metabase.driver :: Registered driver :sqlite.
2016-02-09 16:30:51,616 [INFO ] metabase.driver :: loading driver namespace:  metabase.driver.sqlserver
2016-02-09 16:30:51,630 [DEBUG] metabase.driver :: Registered driver :sqlserver.
2016-02-09 16:30:51,631 [INFO ] metabase.driver :: loading driver namespace:  metabase.driver.sync
2016-02-09 16:30:52,082 [INFO ] metabase.db :: Verifying Database Connection ...
2016-02-09 16:30:57,088 [ERROR] metabase.driver :: Failed to connect to database: Timed out after 5000 milliseconds.
Exception in thread "main" java.lang.AssertionError: Assert failed: Unable to connect to Metabase DB.
(binding [*allow-potentailly-unsafe-connections* true] (require (quote metabase.driver)) ((clojure.core/deref (resolve (quote metabase.driver/can-connect-with-details?))) engine details))
        at metabase.db$verify_db_connection.invoke(db.clj:149)
        at metabase.db$setup_db.doInvoke(db.clj:163)
        at clojure.lang.RestFn.invoke(RestFn.java:421)
        at metabase.core$init_BANG_.invoke(core.clj:130)
        at metabase.core$start_normally.invoke(core.clj:211)
        at metabase.core$_main.doInvoke(core.clj:245)
        at clojure.lang.RestFn.invoke(RestFn.java:397)
        at clojure.lang.AFn.applyToHelper(AFn.java:152)
        at clojure.lang.RestFn.applyTo(RestFn.java:132)
        at metabase.core.main(Unknown Source)
```

--- a/docs/operations-guide/start.md
+++ b/docs/operations-guide/start.md
@@ -98,6 +98,21 @@ If this happens, setting a few JVM options should fix your issue:
 
 Alternatively, you can upgrade to Java 8 instead, which will fix the issue as well.
 
+### Metabase fails to connect to H2 Database on Windows 10
+
+In some situations the Metabase JAR needs to be unblocked so it has permissions to create local files for the application database.
+
+On Windows 10, if you see an error message like
+
+    Exception in thread "main" java.lang.AssertionError: Assert failed: Unable to connect to Metabase DB.
+
+when running the JAR, you can unblock the file by right-clicking, clicking "Properties", and then clicking "Unblock".
+See Microsoft's documentation [here](https://blogs.msdn.microsoft.com/delay/p/unblockingdownloadedfile/) for more details on unblocking downloaded files.
+
+There are a few other reasons why Metabase might not be able to connect to your H2 DB. Metabase connects to the DB over a TCP port, and it's possible
+that something in your `ipconfig` configuration is blocking the H2 port. See the discussion [here](https://github.com/metabase/metabase/issues/1871) for
+details on how to resolve this issue.
+
 
 # Configuring the Metabase Application Database
 

